shell/packages/sandstorm-ui-app-details/app-details-client.js

```diff
@@ -32,7 +32,7 @@ var latestAppManifestForAppId = function (db, appId) {
 
 var getAppTitle = function (appDetailsHandle) {
   var pkg = latestPackageForAppId(appDetailsHandle._db, appDetailsHandle._appId);
-  return SandstormDb.appNameFromPackage(pkg);
+  return pkg && SandstormDb.appNameFromPackage(pkg) || "<unknown>";
```

Comment:
It happens because we are doing subscription joins and the first render might not have an actual `pkg` object yet.